### PR TITLE
Update remote-development.md

### PR DESCRIPTION
**Pull request by ekapner**
### What issues does this PR fix or reference?
@https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008HWSGIA4/view@

### Functionality Before
Documents to try with sample project

### Functionality After
Removes reference to sample project

### CODE REVIEW

**Comment by ekapner** [comment]
@hbc-bindu - once approved, do I merge into develop and that's it? I don't see any other options.

File: docs/_articles/en/user-guide/remote-development.md [hunk 1]
```diff
@@ -30,31 +30,30 @@ To start remote development in dev container, install:
...
+   
+   ![Add Dev Container Config Files](./images/add_dev_container.png)
+   
+1. Select **Salesforce Project** from the list of templates to add `SFDX .devcontainer` folder. The `.devcontainer` folder contains the `devcontainer.json` file that defines how to configure the dev container, the Dockerfile to use, and the extensions to install. 
+   
+   VS Code detects the dev container configuration file and prompts you to reopen the project folder in a container. If the pop-up window disappears, from the Command Palette, run `Remote-Containers: Reopen in Container`.
+   
+   VS code creates a dev container the first time you open the project. After the dev container is built, the project folder in your local system automatically connects and maps to the container, and the side bar shows **Dev Container: Salesforce Project**. The container pre-installs and configures Java, Git, Salesforce CLI, and all other extensions defined in the `devcontainer.json` file.
+   
    ![Dev Coontainer](./images/devcontainer.png)
```
Comment (hbc-bindu) [inline docs/_articles/en/user-guide/remote-development.md:47]: Typo in alt text.

**Comment by hbc-bindu** [comment]
Yes @ekapner

**Comment by ekapner** [comment]
Hi @hbc-bindu - can you do a final review and hopefully approve? Everytime I looked, I found something little I needed to fix. Typos, tagging errors, little nits. But I think it looks good now. Once you approve, I'll merge (seems like a doc maintainer needs to approve, if I'm understanding correctly.

**Comment by lcampos** [comment]
@ekapner @hbc-bindu the changes looked ready so I merged. To clarify, both of you, Stephanie and myself are considered doc maintainers. A doc maintainer approval is required for any changes done to content under the `docs` folder of this project.

**Comment by ekapner** [comment]
> @ekapner @hbc-bindu the changes looked ready so I merged. To clarify, both of you, Stephanie and myself are considered doc maintainers. A doc maintainer approval is required for any changes done to content under the `docs` folder of this project.

@lcampos - thanks for approving and merging. Should we add @clairebianchi as well?

**Comment by ekapner** [comment]
> > @ekapner @hbc-bindu the changes looked ready so I merged. To clarify, both of you, Stephanie and myself are considered doc maintainers. A doc maintainer approval is required for any changes done to content under the `docs` folder of this project.
> 
> @lcampos - thanks for approving and merging. Should we add @clairebianchi as well? Or @praksb for LWT content?

**Pull request merged**